lib/tasks/reports.rake

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+require 'date'
+
+namespace :reports do
+  desc 'creates a csv of all submissions for a grant'
+  task :grant_submissions, [:grant] => [:environment] do |t, args|
+    grant = Grant.where(name: args[:grant]).first
+    submissions = grant.submissions
+
+    csv_file = "#{Rails.root}/tmp/#{grant}_submissions_#{Time.now.strftime("%Y-%m-%d-%H-%M-%S")}.csv"
```

Comment:
Is #{grant} supposed to return grant name? If so, you might want to parameterize it